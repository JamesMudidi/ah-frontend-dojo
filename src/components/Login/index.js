import React from 'react';

const Login = () => (
  <div>
    <h1>Authors&apos; Haven dummy login page</h1>
  </div>
);
export default Login;
