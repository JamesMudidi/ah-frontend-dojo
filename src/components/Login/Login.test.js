import React from 'react';
import { shallow } from 'enzyme';
import Login from 'components/Login';

// test to check if the Login component matches the snapshot
describe('Login Component', () => {
  it('matches the snapshot', () => {
    const component = shallow(<Login />);
    expect(component).toMatchSnapshot();
  });
});
