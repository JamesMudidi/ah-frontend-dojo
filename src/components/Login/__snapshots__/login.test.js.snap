// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component matches the snapshot 1`] = `
<div>
  <h1>
    Authors' Haven dummy login page
  </h1>
</div>
`;
